Adding some classes to help with Integration Tests (#2337)
rsksmart/rskj: rskj-core/src/integrationTest/README.md

## rskj-core integrationTests

This module contains integration tests for the RSKj node, these tests aim to validate core functionalities from the node

> and validate key features from the tool. We takes sometime to be executed and are part of the CI pipeline to ensure the

Reviewer: ```suggestion
and validate key features from the tool. It takes sometime to be executed and is part of the CI pipeline to ensure the
```

RSKj core functionalities are working as expected.

We are trying to improve and increase the coverage of the integration tests, if you have any suggestion or want to 
contribute with new tests, please feel free to open a PR. To make this task easy, some classes were created to help the
creation of nodes inside integration tests scenarios and the validation that they worked as expected.

### Running the integration tests

You will need gradle installed in your machine to run the integration tests. To run the integration tests, execute the 
following command:

```shell

> ./gradlew --no-daemon --stacktrace integrationTest

Reviewer: ```suggestion
./gradlew --stacktrace integrationTest
```
with daemon next executions might run faster. Is there any specific reason for using this flag?
